<launch>
  <arg name="use_rp" default="false"/>

  <node if="$(arg use_rp)" name="rplidarNode"          pkg="rplidar_ros"  type="rplidarNode" output="screen">
  <param name="serial_port"         type="string" value="/dev/ttyUSB1"/>
  <param name="serial_baudrate"     type="int"    value="115200"/><!--A1/A2 -->
  <!--param name="serial_baudrate"     type="int"    value="256000"--><!--A3 -->
  <param name="frame_id"            type="string" value="laser"/>
  <param name="inverted"            type="bool"   value="false"/>
  <param name="angle_compensate"    type="bool"   value="true"/>
  <param name="output_scan"         type="string" value="/up_scan"/>
  </node>

  <node pkg="leo_driving" type="leo_driving_node" name="leo_driving_node" output="screen">
    <param name="Kpy_param" value="1.1"  /> 
    <param name="linear_vel" value="0.2" /> 
    <param name="boundary_percent" value="0.02" /> 
    <param name="side_obstacle_dist" value="0.2" /> 
    <param name="front_obstacle_dist" value="0.1" /> 
    <param name="obastalce_coefficient" value="0.1" /> 
  </node>


</launch>

